lib/app.js

```diff
@@ -42,6 +40,8 @@ app.use('/api', router);
 app.post('/cloud/:datasetId', function(req, res) {
   res.send('ok');
 });
+app.use('/admin', bodyParser.json({limit: '10mb'}), validateSessionMiddleware,
+  require('./app/admin')(authServiceGuid, mbaasApi));
```

Comment:
Using `validateSessionMiddleware` in order to enforce authentication to the data reset endpoint,
`bodyParser` because I think it's required by `validateSessionMiddleware` but not sure